### PR TITLE
Feature/#276 add discord trigger

### PR DESCRIPTION
**Pull request by fmartinou**
Finalization & test of the PR https://github.com/fmartinou/whats-up-docker/pull/276

### CODE REVIEW

File: app/triggers/providers/discord/Discord.js [hunk 1]
```diff
@@ -53,19 +58,20 @@ class Discord extends Trigger {
...
     /**
      * Post a message to discord webhook.
-     * @param text the text to post
+     * @param title the message title
+     * @param bodyText the text to post
      * @returns {Promise<>}
      */
     async sendMessage(title, bodyText) {
-        const uri = `${this.configuration.url}`;
+        const uri = this.configuration.url;
```
Comment (fmartinou) [inline app/triggers/providers/discord/Discord.js:66]: String interpolation is not needed.

File: app/triggers/providers/discord/Discord.js [hunk 2]
```diff
@@ -53,19 +58,20 @@ class Discord extends Trigger {
...
         const body = {
-            username: `${this.configuration.botusername}`,
+            username: this.configuration.botusername,
```
Comment (fmartinou) [inline app/triggers/providers/discord/Discord.js:68]: String interpolation is not needed.

**Comment by fmartinou** [comment]
Hi @adebrouvier ,

Can you review the few changes I made to the `Discord trigger` code, please?
The `travis` build is passing; can you test the Docker image `fmartinou/whats-up-docker:feature__276_add_discord_trigger` and confirm that the trigger is still working as expected?

Thank you :)

**Review by adebrouvier — APPROVED** [review]
LGTM.

Just deployed it to my home server and works like a charm. 🚀 
image

**Pull request merged**